andela/metis-ah
# #160612884 users should be able get details of an article

**What does this PR do?**
Allow users to view details of a single article

**Description of Task to be completed?**

- write tests for the endpoint
- plug in a middleware to ensure that `articleId` is a valid number
- get the following endpoint working: GET `api/v1/articles/:articleId`

**How should this be manually tested?**

Clone the **repository**
Check out the `ft-users-should-be-able-get-details-of-an-article-160612884` branch
Run `npm install` on your local machine to install required Dependencies
Run database migrations using `npm run migrate`
Run `npm test`
_Using Postman_
Start the server using `npm start`

Go to **GET** `localhost:8000/api/v1/articles` with postman
Note: Authentication token is required to access this route. Get that from the login endpoint or signup endpoint

**What are the relevant pivotal tracker stories?**
#160612884
